web/packages/teleport/src/Roles/Roles.tsx

```diff
@@ -93,7 +93,7 @@ export function Roles(props: State) {
               Role-based access control
             </Text>
             <Text typography="subtitle1" mb={3}>
-              Kubernetes and SSH authentication in one place. A Teleport role
+              Fine-grain control over who can and how they access resources. A Teleport role
```

Comment:
- This isn't a complete sentence
- Isn't it _fine-grained_?

Reply:
It's better, but I still think the wording around "who can and how they" can be cleaned up.

Reply:
Please see update from @alexfornuto